pkgs/misc/vim-plugins/default.nix

```diff
@@ -148,6 +148,31 @@ rec {
     dependencies = [];
   };
 
+  LanguageClient-neovim = let
+    LanguageClient-neovim-src = fetchgit {
+      url = "https://github.com/autozimu/LanguageClient-neovim";
+      rev = "fbc46862af7fa254f74f1108149fd0669c46f1ad";
+      sha256 = "1wrrmikriyw8an8hn7240igcaca9a0ykh1j0dfy45kslxkmqkk3r";
+    };
+    LanguageClient-neovim-bin = rustPlatform.buildRustPackage {
+      name = "LanguageClient-neovim-bin";
+      src = LanguageClient-neovim-src;
+
+      cargoSha256 = "0c2sklpvab63a1f1mhcq9abq5m2srkj52ypq7dq44g8ngn2a05ka";
+    };
+  in buildVimPluginFrom2Nix {
+    name = "LanguageClient-neovim-2018-03-06";
+    src = LanguageClient-neovim-src;
+
+    dependencies = [];
+    propogatedBuildInputs = [ LanguageClient-neovim-bin ];
+
+    preFixup = ''
+      substituteInPlace "$out"/share/vim-plugins/LanguageClient-neovim/plugin/LanguageClient.vim \
+        --replace "let l:command = [s:root . '/bin/languageclient']" "let l:command = ['${LanguageClient-neovim-bin}/bin/languageclient']"
+    '';
```

Comment:
This change was not possible with `pkgs/misc/vim-plugins/vim2nix/additional-nix-code`?

Reply:
@Mic92: @johnchildren tried to do it, but failed ([here](https://github.com/NixOS/nixpkgs/issues/33391#issuecomment-372066712)).

Reply:
@garbas ok. let's keep it this way. vimPlugins should get a refactoring anyway. It is pretty hacky.

Reply:
Please also port it to `18.03`.

Reply:
:+1: i agree about the hacky way and the need to refractor it.